ErgoScript-Developer-Course/batch-0/zuli/HW2_zuli/build.sbt

```diff
@@ -0,0 +1,84 @@
+import org.ergoplatform.compiler.ErgoScalaCompiler._
+import org.ergoplatform.playgroundenv.utils.ErgoScriptCompiler
+import org.ergoplatform.playground._
+
+//1. Booleans
+val thisbool = true
+val y = !false
+
+//2. if else statements
+val a=1
+val b=2
+val smallervalue = if (a < b ) a else b
+println(smallervalue)
+
+//Datatypes
+
+//long
+val x = 123L
+println(x)
+
+//Boolean
+val alreadytrue = thisbool
+println(alreadytrue)
+println(y)
+
+//Byte
+val something = Array[Byte](10, -20, 30)
+println(something)
+
+//complex data types
+
+//lists or collections
+val numbers = List(1,2,3,4)
```

Comment:
Again, in ErgoScript use Coll() instead of List or Array:
`val numbers: Coll[Int] = Coll(1,2,3,4)`